README.md

```diff
@@ -1,34 +1,35 @@
-# Packetvisor 3.0
+# Packetvisor 3.0 (Rust)
+This is ported in Rust from C
 
 ## Prerequisite
 
 ### bpftool & XDP-tools (submodules)
-PV 3.0 mainly uses XDP-tools lib and bpftool is used for compiling XDP-tools lib.
+PV 3.0(Rust) mainly uses XDP-tools lib and bpftool is used for compiling XDP-tools lib.
 
 The script, `build.sh`, includes submodule update so that you can do the next step.
 
 ## Build
 You can build PV 3.0 by executing `build.sh`
 
-`sudo sh build.sh`
+`./build.sh`
 
-## Use
-Superuser is needed to execute PV 3.0.
+This script includes allowance of capabilities (CAP_SYS_ADMIN, CAP_NET_ADMIN, CAP_NET_RAW, CAP_DAC_OVERRIDE) to `pv3_rust` so that `sudo` is not needed to run the app.
 
+## Use
+the app, `pv3_rust`, is located at `/target/release`.
 The following is the usage of PV 3.0.
 
-`sudo ./pv <inferface name> <chunk size> <chunk count> <Filling ring size> <RX ring size> <Completion ring size> <TX ring size>`
+`./pv3_rust <inferface name> <chunk size> <chunk count> <Filling ring size> <RX ring size> <Completion ring size> <TX ring size>`
```

Comment:
It would be great if describe rings here
RX/TX rings - obvious, But not other is to first comer

Reply:
I got it. I added description of XSK in README.md